packages/contracts/src/connect-contracts.ts

```diff
@@ -0,0 +1,141 @@
+import { Signer, Contract, providers, ethers } from 'ethers'
+import { Provider } from '@ethersproject/abstract-provider'
+import { getL1ContractData, getL2ContractData } from './contract-data'
+
+export type Network = 'goerli' | 'kovan' | 'mainnet'
+interface L1Contracts {
+  addressManager: Contract
+  canonicalTransactionChain: Contract
+  executionManager: Contract
+  fraudVerifier: Contract
+  xDomainMessenger: Contract
+  ethGateway: Contract
+  multiMessageRelayer: Contract
+  safetyChecker: Contract
+  stateCommitmentChain: Contract
+  stateManagerFactory: Contract
+  stateTransitionerFactory: Contract
+  xDomainMessengerProxy: Contract
+  l1EthGatewayProxy: Contract
+  mockBondManger: Contract
```

Comment:
Is this really the list we want to expose here? We may not need to expose e.g., `mockBondManager` (which has a typo btw) and the factories perhaps?

Reply:
I would say we should expose the following:

```
addressManager
canonicalTransactionChain
stateCommitmentChain
executionManager
fraudVerifier
l1EthGatewayProxy
l1CrossDomainMessengerProxy
mulitMessageRelayer
bondManager
```

Reply:
We definitely should NOT expose the non-proxied contracts since we've been working hard to keep those hidden as to avoid situations where people interact with the implementations instead of the proxies...